projects/1.20/assets/goblins-tyranny/goblins_tyranny/lang/zh_cn.json

```
@@ -0,0 +1,887 @@
+    "item.goblins_tyranny.mug": "哥布林酒",
+    "item.goblins_tyranny.mushroom_hat_boots": "蘑菇帽",
+    "item.goblins_tyranny.mushroom_hat_chestplate": "蘑菇帽",
+    "item.goblins_tyranny.mushroom_hat_helmet": "蘑菇帽",
+    "item.goblins_tyranny.mushroom_hat_leggings": "蘑菇帽",
+    "item.goblins_tyranny.plate": "盘子",
+    "item.goblins_tyranny.populated_tavern.desc": "客满的酒馆",
+    "item.goblins_tyranny.populated_tavern": "音乐唱片",
+    "item.goblins_tyranny.purple_loincloth_boots": "紫色缠腰布",
+    "item.goblins_tyranny.purple_loincloth_chestplate": "紫色缠腰布",
+    "item.goblins_tyranny.purple_loincloth_helmet": "哥布林伪装耳",
+    "item.goblins_tyranny.purple_loincloth_leggings": "紫色缠腰布",
+    "item.goblins_tyranny.red_loincloth_boots": "红色缠腰布",
+    "item.goblins_tyranny.red_loincloth_chestplate": "红色缠腰布",
+    "item.goblins_tyranny.red_loincloth_helmet": "哥布林伪装耳",
+    "item.goblins_tyranny.red_loincloth_leggings": "红色缠腰布",
+    "item.goblins_tyranny.reinforced_prototype_boots": "天跃靴子",
+    "item.goblins_tyranny.reinforced_prototype_chestplate": "天跃胸甲",
+    "item.goblins_tyranny.reinforced_prototype_helmet": "天跃头盔",
+    "item.goblins_tyranny.reinforced_prototype_leggings": "天跃护腿",
+    "item.goblins_tyranny.shaman_goblin_spawn_egg": "哥布林萨满刷怪蛋",
+    "item.goblins_tyranny.upgraded_prototype_boots": "进阶原型靴子",
+    "item.goblins_tyranny.upgraded_prototype_chestplate": "进阶原型胸甲",
+    "item.goblins_tyranny.upgraded_prototype_helmet": "进阶原型头盔",
+    "item.goblins_tyranny.upgraded_prototype_leggings": "进阶原型护腿",
+    "item.goblins_tyranny.wanderer_goblin_spawn_egg": "哥布林游商刷怪蛋",
+    "jei.goblins_tyranny.engineer_craft_info": "可以在任何哥布林村庄内的工程师工作台上制作",
+    "jei.goblins_tyranny.hide_info": "可以向哥布林游商购买，也可以由哥布林猎人掉落",
+    "jei.goblins_tyranny.ingot_info": "可以由哥布林铁匠用金锭、下界合金碎片、工程师碎片锻成",
+    "jei.goblins_tyranny.liquor_info": "可以从一个哥布林村庄中的酒馆调酒师那里买到",
+    "jei.goblins_tyranny.prototype_info": "可以在哥布林工程师的车间找到",
+    "key.categories.goblins": "哥布林的统治",
+    "key.goblins_tyranny.jump_boost": "翔跃",
+    "painting.goblins_tyranny.beer_poster.author": "Paraste",
+    "painting.goblins_tyranny.beer_poster.title": "Beer Poster",
+    "painting.goblins_tyranny.hunters_in_the_forest.author": "Paraste",
+    "painting.goblins_tyranny.hunters_in_the_forest.title": "Hunters In The Forest",
+    "painting.goblins_tyranny.posterbard.author": "Paraste",
+    "painting.goblins_tyranny.posterbard.title": "Poster Bard",
+    "painting.goblins_tyranny.tavern.author": "Paraste",
+    "painting.goblins_tyranny.tavern.title": "Tavern",
+    "painting.goblins_tyranny.the_goblin_king.author": "Paraste",
+    "painting.goblins_tyranny.the_goblin_king.title": "The Goblin King",
+    "subtitles.bard_em11": "哥布林吟游诗人：攻击",
+    "subtitles.bard_music_2": "哥布林吟游诗人：奏乐",
+    "subtitles.bard_music": "哥布林吟游诗人：奏乐",
+    "subtitles.bartender_hey": "哥布林调酒师：问候",
+    "subtitles.bartender_idle": "哥布林调酒师：空闲",
+    "subtitles.blacksmith_no": "哥布林铁匠：否决",

```

```suggestion
    "subtitles.blacksmith_no": "哥布林铁匠：拒绝",
```
Villager disagrees 村民：拒绝